frontend-v2/src/features/data/SubjectGroupForm.tsx

```diff
@@ -46,20 +46,19 @@ const SubjectGroupForm: FC<ISubjectGroupForm> = ({
       onSubmit={onSubmitGroupIDForm}
       padding="1rem"
       sx={{
-        border: "1px solid",
-        borderColor: "primary.light",
         width: "fit-content",
+        display: 'flex',
+        alignItems: 'center'
       }}
     >
-      <InputLabel
-        htmlFor="selected-group"
-        sx={{
-          fontSize: "0.75rem",
-        }}
-      >
-        Change the group ID for selected rows
-      </InputLabel>
-      <Input
+      <Typography variant="caption" sx={{ marginRight: '1rem'}}>
+        {selected?.length} rows selected.
+      </Typography>
+      <Typography sx={{ marginRight: '1rem'}}>
+        New Group ID
+      </Typography>
+      <TextField
+        size='small'
```

Comment:
This text field needs to have an accessible label linked to it.

Reply:
@eatyourgreens Do you mean to use the 'ID' prop for the screen readers accessibility, or the 'LABEL' prop? If label prop, then this would result in this input having two labels - one to the left of the input, one above (mostly, as this input has a prepopulated value of group ID.

Mockup:
![image](https://github.com/user-attachments/assets/7c1a5bcb-4f09-46fa-ac81-e5cbe827684d)
TextField with 'label' prop = test
![image](https://github.com/user-attachments/assets/c2b79be1-3367-45a0-aee7-490af820f583)

Reply:
You want to mark up the 'Group ID' text as a HTML label, and link it to the input with `htmlFor`. That will make the 'Group ID' text clickable in desktop browsers and tappable in mobile browsers. In Firefox, you can also right click the input and inspect accessibility properties, to confirm that the text input has an accessible name for screen readers. 

https://mui.com/material-ui/react-text-field/#accessibility

Reply:
Got it! Updated

Reply:
👍🏻

Reply:
That was my only change request so feel free to merge when you are ready.